### PR TITLE
Added config option to set ip for ipv4 and ipv6

### PR DESCRIPTION
**Pull request by lclc98**
Allows you to set the IP so that it doesn't autodetect each time.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @lclc98, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#627).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: duckdns/config.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
       "certfile": "fullchain.pem",
       "keyfile": "privkey.pem"
     },
+    "ipv4": "",
+    "ipv6": "",
```
Comment (pvizeli) [inline duckdns/config.json]: I don't think they should be as default config around

Comment (lclc98) [reply duckdns/config.json]: I currently have that as the default because not setting the ip would make it autodetect which follows the same behavior.

Comment (pvizeli) [reply duckdns/config.json]: Sure, but you can also use jq for default value. I don't want show this settings default on UI. They can be around if user explicit set it.

File: duckdns/run.sh [hunk 2]
```diff
@@ -11,6 +11,8 @@ LE_DOMAINS=$(jq --raw-output '.domains[]' $CONFIG_PATH)
 LE_UPDATE="0"
 
 # DuckDNS
+IPV4=$(jq --raw-output '.ipv4' $CONFIG_PATH)
+IPV6=$(jq --raw-output '.ipv6' $CONFIG_PATH)
```
Comment (pvizeli) [inline duckdns/run.sh]: `$(jq --raw-output '.ipv6 // empty' $CONFIG_PATH)`

**Pull request merged**